autogen/main/variables.tf.tmpl

```diff
@@ -188,6 +188,12 @@ variable "node_pools_metadata" {
     default-node-pool = {}
   }
 }
+
+variable "resource_usage_export" {
```

Comment:
Instead of switching to a direct object list (which is also a very confusing data type), we should retain the existing `resource_usage_export_dataset_id` variable and add `enable_network_egress_export` and `enable_resource_consumption_export` variables.

Reply:
Ok, thanks for the feedback. I'll update it.